models/reader.js

```diff
@@ -1,16 +1,16 @@
 'use strict';
 module.exports = function(sequelize, DataTypes) {
-  var User = sequelize.define('User', {
+  var Reader = sequelize.define('Reader', {
     name: DataTypes.STRING,
-    surname: DataTypes.STRING,
     email: DataTypes.STRING,
-    password: DataTypes.STRING
+    password: DataTypes.STRING,
```

Comment:
this should also be passwordDigest, you can have the password attribute as a VIRTUAL attribute and a setter.